### ISSUE TITLE
Add limitations section to documentation for Go / pgx

### ISSUE THREAD

**Issue by chriscasano** [cockroachdb/docs#7715]
Chris Casano (chriscasano) commented:

At times, we've directing users to pgx from lib/pq.  However, we don't have documentation around pgx, what it's capabilities are and what is lacking.  An example would be items like this: https://github.com/cockroachdb/cockroach/issues/19603. Most users wouldn't know this limitation until they hit it.  

Jira Issue: DOC-584

**Comment by jseldess** [comment]
@ericharmeling, @vy-ton, passing to you for triage and prioritization.  I have a feeling you're already working on pgx support/docs.

**Comment by vy-ton** [comment]
Yes, Eric has a in-progress [PR](https://github.com/cockroachdb/docs/pull/7698#issuecomment-653215843). @ericharmeling Can you add a limitations section for pgx?

**Comment by ericharmeling** [comment]
> Yes, Eric has a in-progress [PR](https://github.com/cockroachdb/docs/pull/7698#issuecomment-653215843). @ericharmeling Can you add a limitations section for pgx?

Sure thing. Thanks for the notice!

**Comment by ericharmeling** [comment]
@chriscasano 

I can add a "Limitations" section to our [pgx tutorial](https://github.com/cockroachdb/docs/pull/7698) (not yet published). It would be helpful to have a list of said limitations ahead of creating that PR. https://github.com/cockroachdb/cockroach/issues/19603 seems to be a limitation of lib/pq (i.e., a reason for switching over the pgx), rather than a limitation of pgx.

Re: pgx capabilities, I think we should point to pgx documentation (https://github.com/jackc/pgx#features) for a list of the library's capabilities rather than duplicating that information on our docs. I can add a link to that README.

**Comment by chriscasano** [comment]
👍

**Comment by vy-ton** [comment]
> cockroachdb/cockroach#19603 seems to be a limitation of lib/pq (i.e., a reason for switching over the pgx), rather than a limitation of pgx.

@ericharmeling It's not a limitation with pgx but instead CockroachDB's usage with pgx since we don't support the `COPY BINARY`. Probably worth highlighting this so that users using pgx's `COPY` know that it won't work.

**Comment by mjibson** [comment]
We just merged BINARY support, so we should be able to document this better now?
